frame/support/src/traits/validation.rs

```diff
@@ -23,9 +23,11 @@ use sp_runtime::{
 	traits::{Convert, Zero},
 	BoundToRuntimeAppPublic, ConsensusEngineId, Permill, RuntimeAppPublic,
 };
-use sp_staking::SessionIndex;
 use sp_std::prelude::*;
 
+/// Simple index type with which we can count sessions.
+pub type SessionIndex = u32;
```

Comment:
If we really want to avoid duplicated defs, we could move this to a comment deps between frame-support & sp-staking